src/components/Calendar/ScheduleCalendar.js

```diff
@@ -177,10 +177,11 @@ class ScheduleCalendar extends PureComponent {
         calendarHeader.style.marginRight = '0px';
 
         // Set each rbc header and time slot text color to white if the theme is dark
+        // TODO: If theme is auto, by default set to light theme. Need solution
         let rbc_headers = document.getElementsByClassName('rbc-header');
         let rbc_time_slots = document.getElementsByClassName('rbc-label');
-        console.log(rbc_headers);
-        if (isDarkMode()) {
```

Comment:
Why did you replace this call to `getDarkMode`? If it isn't working, I think it's worth modifying `isDarkMode` itself so that it can work here.